src/styles/blocks/main-page.scss

```
@@ -0,0 +1,36 @@
+@import "./header/header.scss";
+@import "./header/nav.scss";
+@import "./main/main.scss";
+@import "./main/for-whom.scss";
+@import "./main/features.scss";
+@import "./main/potr-team.scss";
+
+.main-home {
+  position: relative;
+  background-color: $gray;
+  z-index: 0;
+}
+
+.page-home {
+  position: relative;
+  margin: 0 auto;
```

```suggestion
  margin: 0 auto;
```
No margins for BEM blocks